numba/tests/test_objects.py

```diff
@@ -30,6 +30,12 @@ def delattr_usecase(o):
 
 
 class TestAttributes(TestCase):
+    def test_getattr(self, flags=enable_pyobj_flags):
+        import numba.tests.usecases as uc
+        @njit
+        def f():
+            return uc._GLOBAL_STR
+        self.assertEquals(f(), "abc")
```

Comment:
Puzzled as to why this test is here? This module seems to be related to "objects" but this is compiled under `nopython` mode?

Reply:
Good point. I didn't pay attention to the test file when I reviewed the original code.

Reply:
i think `test_auto_constants.py` is the closest test file.

Reply:
Looks good.